### ISSUE TITLE
Anaconda downloads fail (but url seems correct)

### ISSUE THREAD

**Issue by pescobar** [hpcugent/easybuild-framework#2021]
When trying to install Anaconda doing `eb Anaconda2-4.0.0.eb --debug` I get this error:

```
== 2016-11-23 16:26:24,348 filetools.py:335 WARNING URL http://repo.continuum.io/archive/Anaconda2-4.0.0-Linux-x86_64.sh was not found (HTTP response code 403), not trying again
```

The weird thing is that if you copy&paste that url and try wget it works. 

Maybe the anaconda web servers are checking user-agent to avoid automated downloads?

**Comment by boegel** [comment]
Hmm, not sure what to do with this. If they block, there's nothing much we can/should do?

@jerowe Thoughts on this?

**Comment by jerowe** [comment]
I'm not sure. It worked on my setup. @boegel , didn't work on the robot tests?

If they are blocking, and it would be unfortunate if they are, I suppose we'll all just have to download it.

**Comment by boegel** [comment]
@pescobar can you check whether the change in #2083 fixes this problem for you?

**Comment by boegel** [comment]
fixed in #2083, confirmed by @pescobar
